charts/gardener-extension-provider-packet/templates/configmap-logging.yaml

```diff
@@ -21,15 +21,8 @@ data:
         Parser              kubeapiserverParser
         Reserve_Data        True
 
-    [FILTER]
-        Name                parser
-        Match               kubernetes.*csi-packet-controller*packet-csi*
-        Key_Name            log
-        Parser              kubeapiserverParser
-        Reserve_Data        True
-
     [FILTER]
         Name                rewrite_tag
         Match               kubernetes.*
-        Rule                $tag ^kubernetes\.var\.log\.containers\.(cloud-controller-manager-.+?_.+?_packet-cloud-controller-manager|csi-packet-controller-.+?_.+?_packet-csi) user-exposed.$TAG true
+        Rule                $tag ^kubernetes\.var\.log\.containers\.(cloud-controller-manager-.+?_.+?_packet-cloud-controller-manager) user-exposed.$TAG true
```

Comment:
I wasn't 100% sure about the correctness of the filter syntax after removing the CSI-related stuff (and am not sure how to test it). I'd appreciate if you folks could verify.

Reply:
FMPOV this looks good, @Kristian-ZH or @vlvasilev can you please confirm?